extensions/amp-video-docking/1.0/storybook/Basic.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../../src/preact';
+import {Dock} from '../dock';
+import {Dockable} from '../dockable';
+import {VideoWrapper} from '../../../amp-video/1.0/video-wrapper';
+import {boolean, object, text, withKnobs} from '@storybook/addon-knobs';
+import {withA11y} from '@storybook/addon-a11y';
+
+export default {
+  title: 'Video Dock',
+  component: Dock,
+  decorators: [withA11y, withKnobs],
+};
+
+const Spacer = () => <div style={{height: '100vh'}}></div>;
+
+const renderPlain = (isDocked, dockedStyle) => {
```

Comment:
This is really cool that now anything can be docked :O 

Is it possible for elements that don't play and pause to also specify conditions for when they are dockable?